aixplain/factories/model_factory.py

```diff
@@ -404,3 +404,68 @@ def onboard_model(cls, model_id: Text, image_tag: Text, image_hash: Text, api_ke
         message = "Your onboarding request has been submitted to an aiXplain specialist for finalization. We will notify you when the process is completed."
         logging.info(message)
         return response
+    
+    @classmethod
+    def deploy_huggingface_model(cls, name: Text, hf_repo_id: Text, hf_token: Optional[Text] = None, api_key: Optional[Text] = None) -> Dict:
+        """Onboards and deploys a Hugging Face large language model.
+
+        Args:
+            name (Text): The user's name for the model.
+            hf_repo_id (Text): The Hugging Face repository ID for this model ({author}/{model name}).
+            hf_token (Text, optional): Hugging Face access token. Defaults to None.
+            api_key (Text, optional): Team API key. Defaults to None.
+        Returns:
+            Dict: Backend response
+        """
+        supplier, model_name = hf_repo_id.split("/")
+        deploy_url = urljoin(config.BACKEND_URL, f"sdk/model-onboarding/onboard")
+        if api_key:
+            headers = {"x-aixplain-key": f"{cls.aixplain_key}", "Content-Type": "application/json"}
```

Comment:
should it be api_key instead of cls.aixplain_key?

Reply:
Ah yes! Thanks for the catch!